client/src/ui/deckbuilder/DeckBuilderSection.vue

```diff
@@ -1,26 +1,86 @@
 <template>
   <div class="_deck-builder-section">
-    <div class="column-cnt">
+    <div class="column-cnt"
+         ref="columnContent"
+         @mousedown="mouseDown"
+         @mousemove="mouseMove"
+         @mouseup="mouseUp"
+         >
       <DeckBuilderColumn
           v-for="(column, index) in columns"
+          ref="columns"
           :key="index"
           :column="column"
           :deckIndex="deckIndex"
           :maindeck="maindeck"
           :columnIndex="index"
+          :selectionRectangle="selectionRectangle"
           />
+      <div
+          class="selection"
+          :hidden="selectionRectangle === null"
+          :style="{
+          top: selectionTop,
+          left: selectionLeft,
+          width: selectionWidth,
+          height: selectionHeight,
+        }"></div>
     </div>
   </div>
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
+import Vue, { VueConstructor } from 'vue'
 import DeckBuilderColumn from "./DeckBuilderColumn.vue";
-import { CardColumn } from '../../state/DeckBuilderModule';
+import { CardColumn, CardLocation } from '../../state/DeckBuilderModule';
+import { Point, Rectangle } from "../../util/rectangle";
 
-export default Vue.extend({
+export default (Vue as VueConstructor<Vue & {
```

Comment:
Kudos for figuring out how to get this to work.

Reply:
stack overflow dot org: it's dot com!